switches/README.md

```diff
@@ -0,0 +1,124 @@
+# Switch component
+
+## Switch design specs
+You can find the design specs on [www.decathlon.design](https://www.decathlon.design)
+
+## Usage with Vitamin Theme
+
+By inheriting the Vitamin theme in your app, you just have to use the default SwitchMaterial component or the VitaminSwitch component. 
+
+```kotlin
+implementation 'com.decathlon.android.vitamin:<version>'
+```
+
+```xml
+<com.google.android.material.switchmaterial.SwitchMaterial
+    android:id="@+id/switch"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:checked="true"
+    android:text="My switch component" />
+```
+
+or 
+
+```xml
+<com.decathlon.android.switches.VitaminSwitch
+    android:id="@+id/switch"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:checked="true"
+    android:text="My switch component" />
+```
+
+## Usage in standalone
+
+If you don't want to inherit the full Vitamin theme (and all the Vitamin components) on your entire app, you can use this component in standalone.
+
+### Version 1
+
+Inherit the Base Vitamin theme in your app to have the right colors and fonts and link the Vitamin switch style.
+You can now use the switch component as seen previously.
+
+```kotlin
+implementation 'com.decathlon.android.foundation:<version>'
+implementation 'com.decathlon.android:switches:<version>'
```

Comment:
`com.decathlon.vitamin:switches-vitamin:<version>`

Reply:
Change in all versions

Reply:
Fixed